horace_core/sqw/@Experiment/private/build_from_old_headers_.m

```diff
@@ -106,7 +106,7 @@
 
 % Construct the new header Experiment object
 % update with expdata, which maybe should go in the Experiment constructor
-obj = Experiment([], instruments, samples);
+obj = Experiment(IX_detector_array.empty, instruments, samples);
 obj.expdata = expdata;
```

Comment:
I would allow old interface (user would not be so keen to learn about `IX_detector_array.empty` as `[]` looks much easier ) and make the internal assignment to `IX_detector_array.empty` as this happen